src/Makefile.test.include

```diff
@@ -15,7 +15,10 @@ EXTRA_DIST += \
 	test/data/tx394b54bb.hex \
 	test/data/txcreate1.hex \
 	test/data/txcreate2.hex \
-	test/data/txcreatesign.hex
+	test/data/txcreatesign.hex \
+	test/data/txcreatedata_seq0.hex \
+	test/data/txcreatedata_seq1.hex \
```

Comment:
The three files above were missing from this list, might as well add them; now this list matches the `.hex` files in `test/data` (as is the case with bitcoin). The file below is new.